docs/walkthrough/using-functions.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: Using Functions and Isolates
+nav_order: 12
+parent: Getting Started
+---
+
+## Using Functions and Isolates
+
+Functions exist in the following forms:
+
+- `subflow`
+- `isolated`
+- `reusable`
+- `knative-namespace`
+- `knative-global`
+
+This article demonstrates how each of the available function types is used within a workflow.
+
```

Comment:
This article doesn't demonstrate sub flows or isolated. It's fair to link to other articles for more info, but I think one snippet is still worth having here.